### ISSUE TITLE
Usernames without discriminators given a #0 at the end

### ISSUE THREAD

**Issue by phoebe-leong** [mesalytic/discord-lookup-api#9]
Examples:

https://discordlookup.mesavirep.xyz/v1/user/622004374587179008
https://discordlookup.mesavirep.xyz/v1/user/775475238669516826

can probably implement a simple fix to this because I'm guessing this is mainly a problem on Discord's end.

**Comment by mesalytic** [comment]
This is intentional on Discord’s end as the username migration is not over, so it’s kept as a temporary compatibility fix

Though I could do something on the back-end to remove the #0 when there is no discrim

**Comment by phoebe-leong** [comment]
yeah i thought maybe ``.split()`` or something like it could be implemented when the discriminator is less than four

**Comment by mesalytic** [comment]
not a .split because discord returns the discrim seperately, it’s just a matter of using said discrim data if discrim > 0
